docs/spec.md

```diff
@@ -538,6 +561,20 @@ All integer types explicitly specified (uint32, uint64, etc.) in this section ar
 | _encoding length_ | char[]  | encoding, same encoding that was used for the request |
 | remaining bytes   | uint8[] | response payload                                      |
 
+### Fetch Asset Response
+
+- Response to a [fetch asset](#fetch-asset) request
+- Only supported if the server previously declared the `assets` [capability](#server-info).
+
+| Bytes           | Type             | Description                                                                                                             |
+| --------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| 1               | opcode           | 0x04                                                                                                                    |
+| 4               | uint32           | request id, as given in the corresponding [request](#fetch-asset)                                                       |
+| 1               | uint8            | status enum, `0` for success and `1` for error. Values `>1` are reserved for future use                                 |
+| 4 + N           | uint32 + char[N] | error message, empty if `status == 0`                                                                                   |
```

Comment:
The way these response fields are documented does not match how we've documented this kind of length prefixed response for other kinds of responses:

https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md#service-call-response

Consistency is important for spec sanity so we should pick the style we want. I'd say this PR should follow the status-quo and a separate PR can propose altering the status-quo.